File: Frontend/dgu_laf/lib/model/item_image.dart
class ItemImage {
  final int imageId;
  final int itemId;
  final bool isLost;
  final String imageUrl;

  ItemImage({
    required this.imageId,
    required this.itemId,
    required this.isLost,
    required this.imageUrl,
  });

  factory ItemImage.fromJson(Map<String, dynamic> json) {
    return ItemImage(
      imageId: json['image_id'],
      itemId: json['item_id'],
      isLost: json['is_lost'],
      imageUrl: json['image_url'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'image_id': imageId,
      'item_id': itemId,
      'is_lost': isLost,
      'image_url': imageUrl,
    };
  }
}

File: Backend/dgulaf/src/main/java/com/dgulaf/dgulaf/model/ItemImage.java
package com.dgulaf.dgulaf.model;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "itemimages")
public class ItemImage {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer imageId;

    @Column(nullable = false)
    private Integer itemId;

    @Column(nullable = false)
    private Boolean isLost;

    @Column(nullable = false, length = 255)
    private String imageUrl;
}
